src/index.js

```diff
@@ -52,4 +52,13 @@ process.on('SIGINT', cancelHandler)
 process.on('SIGTERM', cancelHandler)
 
 // Main
-main().then(() => require('./pre'))
+// Boolean inputs are stringified, but manually convert this input just in case that changes in the future.
+const emitTelemetry = core.getInput("emit_telemetry").toString();
```

Comment:
This has been a log established guarantee with Actions, so I think it's safe to omit this part. It will also give you an empty string (vs. `null` or `undefined`) if there is no such input provided. 🤷🏻‍♂️ 

```suggestion
const emitTelemetry = core.getInput("emit_telemetry")
```

I'm also fine with leaving it, though we should chop the semi-colon at the least for formatting consistency. 😊 

```suggestion
// Boolean inputs are stringified, but manually convert this input just in case that changes in the future.
const emitTelemetry = core.getInput("emit_telemetry").toString()
```

Reply:
How'd that semicolon get there? My pinky finger has a code formatting style of its own.